Pipfile

```diff
@@ -56,8 +56,8 @@ raven = "==6.9.0"
 requests = ">=1.2.0"
 social-auth-app-django = ">=2.0.0"
 text-unidecode = "*"
-uwsgi = {version = ">=2.0.0", platform_system = "!= 'Windows'"}
-celery = {version = "*", extras = ["redis"]}
+uwsgi = {version = ">=2.0.0",platform_system = "!= 'Windows'"}
+celery = {version = "*",extras = ["redis"]}
```

Comment:
I would recommend keep the spaces as the official documentation of pipenv has these spaces as shown here https://pipenv.readthedocs.io/en/latest/advanced/#specifying-basically-anything.

Reply:
This is what `pipenv install -d pylint-django` did, I haven't changed that manually.

Reply:
Oh, then I'm fine with it since it is so minor. I just checked this is a bug which is fixed in https://github.com/pypa/pipenv/pull/3362. 😢

Reply:
Looks good then, hope this can make the circle CI pass

Reply:
Since ```pylint-django``` is already in the Pipfile, usually ```pipenv lock``` is enough. If you would like to re-lock without cache, ```pipenv lock --clear``` will do that.